.github/site/styles.css

```
@@ -1,16 +1,19 @@
-  font-family: "Inter Tight", "Arial Narrow", ui-sans-serif, system-ui, sans-serif;
+  --paper: #edf1ef;
+  --paper-muted: #9ba4a4;
+  --ink: #151819;
+  --surface: #1c2021;
+  --surface-raised: #252a2b;
+  --line: rgba(237, 241, 239, 0.14);
+  --line-bright: rgba(237, 241, 239, 0.3);
+  --mark: #66a9c2;
+  --mark-bright: #8bc1d4;
+  --green: #78ad91;
+  font-family: "Manrope Variable", ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)